drivers/overlay/overlay.go

```diff
@@ -898,9 +898,6 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 	// if we are doing a readOnly mount, and there is only one lower
 	// We should just return the lower directory, no reason to mount.
 	if !readWrite && d.options.mountProgram == "" {
-		if len(absLowers) == 0 {
-			return path.Join(dir, "empty"), nil
-		}
 		if len(absLowers) == 1 {
```

Comment:
do we also need to take care of the `len(absLowers) == 1` case?  Could happen that the image has two layers and we would hit this case and there is no mount happening?

Reply:
Yes I created an image with a single lower layer, and it mounted the wrong thing.  Good catch.  I just removed both special cases and now we will always mount the image.